### PR TITLE
Release Notes for 2023.06.2 (#4240)

### PR DESCRIPTION
**Pull request by timhowie-okta**
## Description:
- **What's changed?** Added entries for Release Notes for 2023.06.2
- **Is this PR related to a Monolith release?** Yes. 2023.06.2

### Resolves:

* [OKTA-621463](https://oktainc.atlassian.net/browse/OKTA-621463)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/f714bf0948208e89d751563a2eaf5d3fc52af7b7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 74 | 349 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/1b76de14-a36a-4794-997a-8735c4e2df90) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/f714bf0948208e89d751563a2eaf5d3fc52af7b7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 76 | 262 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/92268241-d419-44f1-83f6-6f51f2c5a33e) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.754c0210-5829-4f89-9815-fa8020f46728?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Review by isaacdurham-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md [hunk 1]
```diff
@@ -6,6 +6,23 @@ title: Okta Identity Engine API Products release notes 2023
...
+| [Bugs fixed in 2023.06.2](#bugs-fixed-in-2023-06-2) | July 6, 2023 |
+
+#### New IdP permissions for custom admin roles is EA in Preview
+
+Admins can now leverage new Identity Provider management permissions when creating custom admin roles. These permissions allow more precise access control and reinforce the principle of least privilege. See [Roles](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Role/).
+
+#### Bugs fixed in 2023.06.2
+
+* The `max_age=0` property wasn't treated the same as `prompt=login` for OAuth 2.0 `/authorize` requests. (OKTA-588559)
+* A delete session request (/api/v1/sessions/me) didn't clear the session cookie (`sid`). (OKTA-620986)
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:23]: Traditionally, we put the api call in code font: `/api/v1/sessions/me`

Comment (timhowie-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md:23]: Thanks Brian - I've updated the formatting..

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx Scorecards

Acrolinx checked the content changed here.

Please review the scorecards for each file and make the recommended
improvements.

| Article | Score | Issues | Scorecard | Status |
| ------- | ----- | ------ | --------- | ------ |
| [packages/@okta/vuepress-site/docs/release-notes/2023-okta-identity-engine/index.md](https://github.com/okta/okta-developer-docs/blob/ee23e1d43987b52bd659725f8436dc60505181e2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023-okta-identity-engine%2Findex.md) | 74 | 349 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/4a086595-388b-48f2-a5cf-e8fa5b00de62) | [:white_check_mark:](# "Successfully processed") |
| [packages/@okta/vuepress-site/docs/release-notes/2023/index.md](https://github.com/okta/okta-developer-docs/blob/ee23e1d43987b52bd659725f8436dc60505181e2/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Frelease-notes%2F2023%2Findex.md) | 76 | 262 | [→](https://okta.acrolinx.cloud/api/v1/checking/scorecards/46b63248-371c-4558-807f-5372173c51b0) | [:white_check_mark:](# "Successfully processed") |

Successfully checked 2 of 2 documents.
See summary in [Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.8067599b-7dd3-41a2-9920-1037f6a07380?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

Reopen the pull request or push new changes to check again.

Depending on the Acrolinx server configuration, the
links expire after some time and you must have a login for the
Acrolinx server to access them again.

**Pull request merged**